packages/calcite-components/src/components/action/action.tsx

```diff
@@ -338,10 +357,15 @@ export class Action extends LitElement implements InteractiveComponent, FormOwne
 
     return (
       <button
-        aria-controls={indicator ? indicatorId : null}
-        ariaBusy={loading}
-        ariaLabel={ariaLabel}
-        ariaPressed={active}
+        ariaBusy={this.aria?.ariaBusy ?? loading}
+        ariaControlsElements={ariaControlsElements}
+        ariaDescribedByElements={this.aria?.ariaDescribedByElements}
+        ariaExpanded={this.aria?.ariaExpanded}
+        ariaHasPopup={this.aria?.ariaHasPopup}
+        ariaLabel={this.aria?.ariaLabel ?? ariaLabel}
```

Comment:
Should we allow overrides for ARIA props when the component exposes an API for that behavior?

Reply:
Fixed